hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/scm/GetScmRatisStatusSubcommand.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.hadoop.ozone.admin.scm;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.hdds.scm.client.ScmClient;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "listratisstatus",
```

Comment:
Name seems a bit verbose. Let me find a syntax to align with OM HA

Reply:
I think both this one and OM HA `getserviceroles` can be improved.

Reply:
@adoroszlai 

What kind of improvement OM HA `getserviceroles` you are thinking of? I am happy to make a separate PR for that.

Reply:
`status` or `roles` would probably be enough to indicate the goal of the subcommand, something like

```
ozone admin (om|scm) status
```

Reply:
Thanks. I will adopt `ozone admin scm status` in this PR and I will send another PR for ``ozone admin om status`

Reply:
ozone admin om getserviceroles -id=<<service id>>

This is what OM does. I have my +1 on ozone admin (om|scm) roles. Status is more like health check.

Reply:
I am ok with both. 

@adoroszlai  what do you think?

Reply:
I think it depends on whether you want to keep this command specific to roles, or may extend the same command in the future with other status info.  Probably "roles" is better now, and "status" can be either a separate command or another alias later.

Reply:
that makes sense. Indeed `status` sounds more like a health check and can carry much more information.

Consider `OM` is also actually getting `roles`. We can start from `roles`